index.html

```diff
@@ -1899,65 +1894,60 @@ <h3>Status</h3>
-<a>subject</a>, but there is no limit to the number of <a>subjects</a> or
-<a>issuers</a> in the data. The aggregation of information from multiple
-<a>verifiable credentials</a> is a typical use of
-<a>verifiable presentations</a>.
+<a>Verifiable presentations</a> MAY be used to aggregate information from
+multiple <a>verifiable credentials</a>.
         </p>
         <p>
-<a>Verifiable presentations</a> SHOULD be extremely short-lived, and
-bound to a challenge provided by a <a>verifier</a>. Details for accomplishing
-this depend on the securing mechanism, the transport protocol, and
-<a>verifier</a> policies. Unless additional requirements are defined by the
-particular securing mechanism or embedding protocol, a <a>verifier</a> cannot
-generally assume that the <a>verifiable presentation</a> has any correlation
-with the presented <a>verifiable credentials</a>.
+<a>Verifiable presentations</a> SHOULD be extremely short-lived, and bound to a
+challenge provided by a <a>verifier</a>. Details for accomplishing this depend
+on the securing mechanism, the transport protocol, and <a>verifier</a> policies.
+Unless additional requirements are defined by the particular securing mechanism
+or embedding protocol, a <a>verifier</a> cannot generally assume that the
+<a>verifiable presentation</a> has any correlation with the presented
+<a>verifiable credentials</a>.
         </p>
         <p>
-A <a>verifiable presentation</a> is typically composed of the following
-properties:
+The following properties are defined for a <a>verifiable presentation</a>:
         </p>
 
         <dl>
           <dt><var>id</var></dt>
           <dd>
-The <code>id</code> <a>property</a> is optional and MAY be used to provide a
-unique identifier for the <a>presentation</a>. For details related to the use of
-this property, see Section <a href="#identifiers"></a>.
+The <code>id</code> <a>property</a> is optional. It MAY be used to provide a
+unique identifier for the <a>verifiable presentation</a>. For details related to
+the use of this property, see Section <a href="#identifiers"></a>.
           </dd>
           <dt><var>type</var></dt>
           <dd>
-The <code>type</code> <a>property</a> is required and expresses the
-type of <a>presentation</a>, such as <code>VerifiablePresentation</code>. For
+The <code>type</code> <a>property</a> MUST be present. It is used to express the
+type of <a>verifiable presentation</a>. The value of this property MUST be
+<code>VerifiablePresentation</code>, but additional types MAY be included. For
 details related to the use of this property, see Section <a href="#types"></a>.
           </dd>
           <dt><var>verifiableCredential</var></dt>
           <dd>
-If present, the value of the <code>verifiableCredential</code> <a>property</a>
-MUST be constructed from one or more <a>verifiable credentials</a>, or of data
+The <code>verifiableCredential</code> <a>property</a> MAY be present. The value
```

Comment:
if its present, can it be empty?

Reply:
What is better:
1. required and can be empty, or
2. not required but has to have at least one value?

Reply:
2 is better... this is a perfect case of "many different JSONs" produce the same "graph"... and proof mechanisms might not preserve all of them the same way...

We have the exact same problem come up with `credentialSubject`.

we said it MUST be present and it MUST NOT be empty.

Doing that for Verifiable Presentations would destroy "DID Auth" which use data integrity proofs to prove possession of a private key for a holder, but does not communicate any credentials.

So my reasoning for 2 is to allow for that behavior to conintue to be legal, while removing equivalent JSON syntax for the same RDF Graph.

Reply:
excellent. 2 is how it's currently worded, so I won't make any changes.